moabb/evaluations/metasplitters.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from sklearn.model_selection import (
+    BaseCrossValidator,
+    StratifiedShuffleSplit,
+)
```

Comment:
You need a header here too, just to explain the whole idea of the meta-splitters